### PR TITLE
Blog - Cross-Compiling for Raspberry Pi: Getting Started and Troubleshooting

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
@balapriyac [Preview Link](https://64640373173fe5269caae38c--ephemeral-narwhal-3ac3b2.netlify.app/blog/cross-compiling-raspberry-pi/)

A bit concerned about code blocks filenames

File: blog/_posts/2023-06-21-cross-compiling-raspberry-pi.md [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+## Setting Up a Cross-Compiling Environment
+
+![Setting Up]({{site.images}}{{page.slug}}/setup.png)\
+
+Before you get started with this tutorial, make sure you have the following:
+
+* **A machine running Ubuntu:** This is where you'll compile the program and is often referred to as the host machine. Note that although it's possible to cross-compile from any Linux distribution, for simplicity, this article will use some Debian- and Ubuntu-based tools. If you don't have an Ubuntu machine, it's recommended to spin up an Ubuntu virtual machine (VM) to follow along. This tutorial was tested with Ubuntu 22.04.
+
+* **A Raspberry Pi:** This should be connected to the local network and accessible using the Secure Shell Protocol (SSH) from the host machine. This article was tested on a Raspberry Pi 4B running Raspbian 10. If you have another board, such as a Raspberry Pi Zero, you need to tweak the commands appropriately. The Raspberry Pi used in this article was available in the local network with `pi.local` hostname and had a `pi` user.
+
```
Comment (balapriyac) [inline blog/_posts/2023-06-21-cross-compiling-raspberry-pi.md:33]:
```suggestion
Before you get started with this tutorial, make sure you have the following:

* **A machine running Ubuntu**: This is where you'll compile the program and is often referred to as the host machine. Note that although it's possible to cross-compile from any Linux distribution, for simplicity, this article will use some Debian- and Ubuntu-based tools. If you don't have an Ubuntu machine, it's recommended to spin up an Ubuntu virtual machine (VM) to follow along. This tutorial was tested with Ubuntu 22.04.

* **A Raspberry Pi**: This should be connected to the local network and accessible using the Secure Shell Protocol (SSH) from the host machine. This article was tested on a Raspberry Pi 4B running Raspbian 10. If you have another board, such as a Raspberry Pi Zero, you need to tweak the commands appropriately. The Raspberry Pi used in this article was available in the local network with `pi.local` hostname and had a `pi` user.

```

**Review by balapriyac — CHANGES_REQUESTED** [review]
This looks great. Just a small formatting edit.

**Comment by msali123** [comment]
@balapriyac [Changes done](https://646b1c770fe5f8201fadcfb9--ephemeral-narwhal-3ac3b2.netlify.app/blog/cross-compiling-raspberry-pi/)

**Review by jalletto — APPROVED** [review]

**Pull request merged**